Bottom: 0e63b6bfff4cc484c3e8420d14abb4993180f970
Top:    aa6c091b6f6a84e3f2b9081b4842b40f00d57924
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-12 01:15:50 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index ed10b50..79a914a 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
